pygpu/gpuarray.pxd

```diff
@@ -142,6 +142,7 @@ cdef extern from "gpuarray/array.h":
 
     bint GpuArray_CHKFLAGS(_GpuArray *a, int fl)
     bint GpuArray_ISONESEGMENT(_GpuArray *a)
+    bint GpuArray_IS_C_CONTIGUOUS(_GpuArray *a)
```

Comment:
Why we don't need it in the python module? I understand that currently it won't be used, but if someone make a new Theano op and need this, I don't want him to need to make change to libgpuarray.